src/components/Forms/AddServerForm.vue

```diff
@@ -76,6 +76,14 @@ export default Vue.extend({
     async connectToServer(): Promise<void> {
       this.loading = true;
 
+      if (!this.serverUrl.match(/^https?:\/\//)) {
+        this.serverUrl = 'http://' + this.serverUrl;
+      }
+
+      if (!this.serverUrl.match(/:[0-9]+$/)) {
+        this.serverUrl = this.serverUrl + ':8096';
+      }
+
```

Comment:
Mmm. 
_**When this PR will be completed**_, I would like to use this fallback logic in that of the [Default Servers](https://github.com/jellyfin/jellyfin-vue/pull/1698) (moving it to a utils function) to apply it also to the addresses specified in the environment variable.

But still think we can always skip the port fallback  :)

Reply:
I don't think you should interpret URLs from the env var too much, _for me_ they should be as standard as possible

Reply:
If you think better I can take it off.

I did this to provide an adequate warn, if for example you provide a list of URLs one of which has a typo.

Only to facilitate the finding of the specific occurrence, indicating the exact problematic address.

It seemed useful to me, didn't you think @ThibaultNocchi ?